elliottlib/cli/advisory_drop_cli.py

```diff
@@ -24,13 +24,17 @@ def advisory_drop_cli(advisory):
         click.echo(f'Advisory {advisory} is already dropped')
         return
 
+    # move advisory status to NEW_FILES
+    if adv.errata_state != "NEW_FILES":
+        adv.setState("NEW_FILES")
+        adv.commit()
+
     # Remove bugs and builds before dropping the advisory
+    adv = Erratum(errata_id=advisory)
```

Comment:
I don't think we need this, commit() will probably do an internal refresh. can you test without this?